code/modules/mob/living/carbon/human/species_types/corvid.dm

```diff
@@ -0,0 +1,34 @@
+/datum/species/corvid
+	name = "Corvid"
+	id = "corvid"
+	say_mod = "sings"
```

Comment:
"sings" isnt a great idea due to the fact that it could be taken out of context as "consistant singing". A better idea would be more bird-like say mods.